src/rpc/rawtransaction.cpp

```diff
@@ -295,35 +304,43 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
 
             "\nResult (if verbose > 0):\n"
             "{\n"
-            "  \"in_active_chain\": b,   (bool) Whether specified block is in the active chain or not (only present with explicit \"blockhash\" argument)\n"
-            "  \"hex\" : \"data\",       (string) The serialized, hex-encoded data for 'txid'\n"
-            "  \"txid\" : \"id\",        (string) The transaction id (same as provided)\n"
-            "  \"authdigest\" : \"id\",  (string) The transaction's auth digest. For pre-v5 transactions this will be ffff..ffff\n"
-            "  \"size\" : n,             (numeric) The transaction size\n"
-            "  \"version\" : n,          (numeric) The version\n"
-            "  \"locktime\" : ttt,       (numeric) The lock time\n"
-            "  \"expiryheight\" : ttt,   (numeric, optional) The block height after which the transaction expires\n"
-            "  \"vin\" : [               (array of json objects)\n"
+            "  \"in_active_chain\": b,        (bool) Whether specified block is in the active chain or not (only present with explicit \"blockhash\" argument)\n"
+            "  \"hex\" : \"data\",              (string) The serialized, hex-encoded data for 'txid'\n"
+            "  \"txid\" : \"id\",               (string) The transaction id (same as provided)\n"
+            "  \"authdigest\" : \"id\",         (string) The transaction's auth digest. For pre-v5 transactions this will be ffff..ffff\n"
+            "  \"size\" : n,                  (numeric) The transaction size\n"
+            "  \"overwintered\" : b,          (bool, optional) Whether the overwintered flag is set\n"
+            "  \"version\" : n,               (numeric) The version\n"
+            "  \"versiongroupid\" : \"hex\",    (string, optional) The version\n"
+            "  \"locktime\" : ttt,            (numeric) The lock time\n"
+            "  \"expiryheight\" : ttt,        (numeric, optional) The block height after which the transaction expires\n"
+            "  \"vin\" : [                    (array of json objects)\n"
             "     {\n"
-            "       \"txid\": \"id\",    (string) The transaction id\n"
+            "       \"coinbase\": b,     (bool, optional) The coinbase script\n"
+            "       \"txid\": \"id\",      (string) The transaction id\n"
```

Comment:
This is incorrect (`coinbase` is a script, not a bool). It is also a bit misleading, because if `coinbase` is present, then the only other field present is `sequence`. Let's have two separate object definitions instead:
```cpp
            "     {                    (coinbase transactions)\n"
            "       \"coinbase\": b,     (string) The coinbase scriptSig as hex\n"
            "       \"sequence\": n      (numeric) The script sequence number\n"
            "     },\n"
            "     {                    (non-coinbase transactions)\n"
            "       \"txid\": \"id\",      (string) The transaction id\n"
```